src/ListboxOption.tsx

```diff
@@ -50,8 +50,9 @@ export class ListboxOption extends Component<Props, State> {
         onMouseDown={this.handleClick}
         onMouseMove={this.handleMouseMove}
         ref={el => this.ownRef = el}
+        role="option"
         tabIndex={-1}
-        {...isSelected ? { "aria-selected": true, role: "option" } : {}}
+        {...isSelected ? { "aria-selected": true } : {}}
```

Comment:
Was having the role only on the selected option intentional? @mitchbne

Reply:
Ahh no it wasn't, great pickup!